e2e-tests/constants/field-ids/insurance/your-buyer/index.js

```diff
@@ -15,6 +15,8 @@ export const YOUR_BUYER = {
   CONNECTION_WITH_BUYER_DESCRIPTION: 'connectionWithBuyerDescription',
   TRADED_WITH_BUYER: 'exporterHasTradedWithBuyer',
   OUTSTANDING_PAYMENTS: 'outstandingPayments',
+  TOTAL_OUTSTANDING: 'totalOutstanding',
+  AMOUNT_OVERDUE: 'amountOverdue',
```

Comment:
What is the "outstanding total" and "amount "for"? In terms of data quality/clarity I think this needs to be more specific, maybe 

```
totalOutstandingPayments
overduePaymentsAmount /  totalOverduePayments
```

What do you think?

Reply:
done